src/main/java/com/puppycrawl/tools/checkstyle/checks/FinalParametersCheck.java

```diff
@@ -19,7 +19,7 @@
 
 package com.puppycrawl.tools.checkstyle.checks;
 
-import java.util.Set;
+import java.util.BitSet;
```

Comment:
Has it been discussed about the performance difference between Set and BitSet?

Reply:
BitSet is very effective for a small zero-based set of integers like an enum. The `get` method is basically
```java
boolean get(int bitIndex) {
  int wordIndex = bitIndex >> 64;
  return words[wordIndex] & (1L << bitIndex)) != 0;
}
```
The `set` method is simple as `get`.

I doubt that there is any *significant* benefit from this, but in general, `BitSet` is faster and more efficient in terms of memory consumption.